### PR TITLE
DOC-4710: Link fixes

### PR DESCRIPTION
**Pull request by AnnaK-R3**

### CODE REVIEW

File: content/en/platform/corda/4.10/enterprise/node/operating/shell.md [hunk 1]
```diff
@@ -507,7 +507,7 @@ Use the different flow commands available to make changes on the ledger. You can
 
 #### Query flow data
 
-The shell can be used to query flow data. For more information on the types of data that can be queried and instructions for doing so, see the documentation on [Querying flow data](../../4.8/enterprise/node/operating/querying-flow-data.html#querying-flow-data-via-the-node-shell).
+The shell can be used to query flow data. For more information on the types of data that can be queried and instructions for doing so, see the documentation on [Querying flow data](querying-flow-data..html#querying-flow-data-via-the-node-shell).
```
Comment (nadinequinr3) [inline content/en/platform/corda/4.10/enterprise/node/operating/shell.md:510]:
```suggestion
The shell can be used to query flow data. For more information on the types of data that can be queried and instructions for doing so, see the documentation on [Querying flow data](querying-flow-data.html#querying-flow-data-via-the-node-shell).
```

File: content/en/platform/corda/4.10/enterprise/operations/monitoring-logging/monitoring-logging-intro.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ To understand the information in the logs, which data is important, and what can
 
 ## Monitoring - the basics
 
-You can monitor the health and performance of your node [using tools such as Jolokia and Graphite](../../node/operating/monitoring-logging/overview.html#node-monitoring). Whenever using these tools, you should follow best-practice steps such as setting up TCP checks for the ports the node communicates on, database health checks etc.
+You can monitor the health and performance of your node [using tools such as Jolokia and graphite. Whenever using these tools, you should follow best-practice steps such as setting up TCP checks for the ports the node communicates on, database health checks etc.
```
Comment (nadinequinr3) [inline content/en/platform/corda/4.10/enterprise/operations/monitoring-logging/monitoring-logging-intro.md:50]:
```suggestion
You can monitor the health and performance of your node using tools such as Jolokia and graphite. Whenever using these tools, you should follow best-practice steps such as setting up TCP checks for the ports the node communicates on, database health checks etc.
```

File: content/en/platform/corda/4.10/enterprise/node/operating/shell.md (hunk 1 above)
Comment (paulmoloneyr3) [inline content/en/platform/corda/4.10/enterprise/node/operating/shell.md:510]:
```suggestion
The shell can be used to query flow data. For more information on the types of data that can be queried and instructions for doing so, see the documentation on [Querying flow data](querying-flow-data.html#querying-flow-data-via-the-node-shell).
```

File: content/en/platform/corda/4.10/enterprise/operations/monitoring-logging/monitoring-logging-intro.md (hunk 2 above)
Comment (paulmoloneyr3) [inline content/en/platform/corda/4.10/enterprise/operations/monitoring-logging/monitoring-logging-intro.md:50]: Think this should be upper case even if their logo is stylised as lower case?

**Review by nadinequinr3 — CHANGES_REQUESTED** [review]

**Review by paulmoloneyr3 — CHANGES_REQUESTED** [review]

File: content/en/platform/corda/4.10/enterprise/operations/monitoring-logging/monitoring-logging-intro.md (hunk 2 above)
Comment (AnnaK-R3) [reply content/en/platform/corda/4.10/enterprise/operations/monitoring-logging/monitoring-logging-intro.md]: Hmm, not sure, if you scroll down the page, it's lower case in our docs so I wanted to stay consistent.

**Review by nadinequinr3 — APPROVED** [review]

**Pull request merged**